fuse_constraints/include/fuse_constraints/normal_prior_orientation_3d_euler_cost_functor.h

```diff
@@ -105,7 +106,7 @@ class NormalPriorOrientation3DEulerCostFunctor
 
     for (size_t i = 0; i < axes_.size(); ++i)
     {
-      T angle;
+      T angle = T(0);
```

Comment:
This is here to satisfy compiler warnings about `angle` potentially not being defined after the introduction of `ErrorHandler` meant that code that actually uses `angle` could be reached without `angle` being guaranteed to have been set. Recall that the `ErrorHandler` call is not always guaranteed to throw an exception